src/Monomer/Graphics/FontManager.hs

```diff
@@ -109,9 +110,9 @@ loadFont ctx fonts fontDef = do
     then return $ name : fonts
     else putStrLnErr ("Failed to load font: " ++ T.unpack name) >> return fonts
   where
-    name = fontDef ^. fontName
-    createFont FontDefFile{} = fmCreateFont ctx name (fontDef ^. fontPath)
-    createFont FontDefMem{} = fmCreateFontMem ctx name (fontDef ^. fontBytes)
+    name = fontDef ^. L.name
```

Comment:
Looks good!

Just to be clear: if you prefer your previous version with the records, go ahead with it; it works well for me.